Map.Api/Controllers/TripController.cs

```diff
@@ -74,12 +72,15 @@ public class TripController : ControllerBase
     [ProducesResponseType(typeof(Error), StatusCodes.Status401Unauthorized)]
     [ProducesResponseType(typeof(Error), StatusCodes.Status403Forbidden)]
     [ProducesResponseType(typeof(Error), StatusCodes.Status500InternalServerError)]
-    public async Task<IActionResult> AddTrip([FromBody] AddTripDto addTripDto)
+    public async Task<ActionResult<TripDto>> AddTrip([FromBody] AddTripDto addTripDto)
     {
         ValidationResult validationResult = await _addTripValidator.ValidateAsync(addTripDto);
         if (!validationResult.IsValid)
             return BadRequest(validationResult.Errors.Select(e => new Error(e.ErrorCode, e.ErrorMessage)));
 
+        if (addTripDto.BackgroundPicturePath is null)
+            addTripDto.BackgroundPicturePath = "https://www.voyageursdumonde.fr/voyage-sur-mesure/magazine-voyage/showphoto/1357/0";
```

Comment:
mettre l'url dans une constante appelé default_picture

Reply:
no need these temporary it will be removed in no time